test/test_train_imagenet.py

```diff
@@ -73,6 +75,11 @@ def get_model_property(key):
   return MODEL_PROPERTIES.get(FLAGS.model, MODEL_PROPERTIES['DEFAULT'])[key]
 
 
+def add_scalar_to_summary(summary_writer, metric_name, y_value, x_value):
```

Comment:
Move this to `test_utils.py`.
Also, what are those x,y values?

Reply:
Moved and renamed x,y to more closely match the documentation: https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_scalar

In my mind, each scalar is an x,y coordinate in the Tensorboard graph. But hopefully this new naming is more clear